articles/search/search-security-overview.md

```diff
@@ -166,16 +166,18 @@ Currently, the only external resource that a search service writes customer data
 
 ### Exceptions to data residency commitments
 
-Although customer data isn't stored outside of your region, object names (considered as customer data), will appear in the telemetry logs used by Microsoft Support to troubleshoot your service issues. Telemetry logs include names of indexes, indexers, data sources, skillsets, containers, and key vault store.
+Non-personal data received from an organization, or generated from the use of the product, such as component names, will be stored and processed outside your selected region or location. This data will appear in the telemetry logs used by Microsoft to provide support for the service. Component names include names of indexes, indexers, data sources, skillsets, containers, and key vault store.
```

Comment:
"customer data" is defined in the data protection addendum.  I don't know what "non-personal" data means, but I think you need to use the term "Customer data" here.   Also, we don't use the term "components" -- we use the term "objects" when referring to the things that are created on a search service.

Here is the official definition --> “Customer Data” means all data, including all text, sound, video, or image files, and software, that are provided to Microsoft by, or on behalf of, Customer through use of the Online Service. Customer Data does not include Professional Services Data